src/workflows-app/SubmissionConfig.js

```diff
@@ -120,6 +125,32 @@ export const BaseSubmissionConfig = (
     [methodId, signal]
   );
 
+  const loadCbasSubmissionLimits = useCallback(
+    async (cbasUrlRoot) => {
+      try {
+        const capabilities = await Ajax(signal).Cbas.capabilities(cbasUrlRoot);
+        if (capabilities) {
+          const newCapabilities = {
+            maxWorkflows: capabilities['submission.limits.maxWorkflows']
+              ? capabilities['submission.limits.maxWorkflows']
+              : cbasSubmissionLimits.maxWorkflows,
+            maxInputs: capabilities['submission.limits.maxInputs'] ? capabilities['submission.limits.maxInputs'] : cbasSubmissionLimits.maxInputs,
+            maxOutputs: capabilities['submission.limits.maxOutputs'] ? capabilities['submission.limits.maxOutputs'] : cbasSubmissionLimits.maxOutputs,
```

Comment:
Yes if do introduce new capabilities they would need to be added as and when they are needed. Since L48 defines submission limits I don't think any new capability not related to submission would need to be added there.